### PR TITLE
Spark checkpoint docs

### PR DESCRIPTION
**Pull request by skonto**
Some basic stuff

### CODE REVIEW

**Comment by skonto** [comment]
@ArtRand some light docs.

File: docs/hdfs.md [hunk 1]
```diff
@@ -21,6 +21,10 @@ For more information, see [Inheriting Hadoop Cluster Configuration][8].
 
 For DC/OS HDFS, these configuration files are served at `http://<hdfs.framework-name>.marathon.mesos:<port>/v1/connection`, where `<hdfs.framework-name>` is a configuration variable set in the HDFS package, and `<port>` is the port of its marathon app.
 
+### Spark Checkpointing
+
+In order to use spark with cehckpointing make sure you follow the instructions [here](https://spark.apache.org/docs/latest/streaming-programming-guide.html#checkpointing) and use an hdfs directory as the checkpoint directory. In the future we plan to support checkpointing at the driver level with local persistent volumes.
```
Comment (ArtRand) [inline docs/hdfs.md]: @skonto This is great. Could you add an example? Also, please remove `In the future we plan to support checkpointing at the driver level with local persistent volumes.` although I like the optimism, let's put this in _when_ we have driver level checkpointing.

**Comment by skonto** [comment]
@ArtRand pls review.

File: docs/hdfs.md [hunk 2]
```diff
@@ -21,6 +21,16 @@ For more information, see [Inheriting Hadoop Cluster Configuration][8].
 
 For DC/OS HDFS, these configuration files are served at `http://<hdfs.framework-name>.marathon.mesos:<port>/v1/connection`, where `<hdfs.framework-name>` is a configuration variable set in the HDFS package, and `<port>` is the port of its marathon app.
 
+### Spark Checkpointing
+
+In order to use spark with checkpointing make sure you follow the instructions [here](https://spark.apache.org/docs/latest/streaming-programming-guide.html#checkpointing) and use an hdfs directory as the checkpointing directory. For example:
+```
+val checkpointDirectory = hdfs://hdfs/checkpoint
```
Comment (susanxhuynh) [inline docs/hdfs.md]: Add quotes around the string: `"hdfs://hdfs/checkpoint"`

File: docs/hdfs.md [hunk 3]
```diff
@@ -87,7 +97,7 @@ Submit the job with the ticket:
 ```$bash
     dcos spark run --submit-args="\
     --kerberos-principal hdfs/name-0-node.hdfs.autoip.dcos.thisdcos.directory@LOCAL \
-    --tgt-secret-path /__dcos_base64__tgt 
+    --tgt-secret-path /__dcos_base64__tgt
```
Comment (susanxhuynh) [inline docs/hdfs.md]: Add a back-slash ("\") at end of this line

**Comment by skonto** [comment]
@susanxhuynh pls review, I think its clear for merge.

**Pull request merged**